include/nbl/ui/KeyCodes.h

```diff
@@ -1,16 +1,19 @@
 
 namespace nbl::ui
 {
-	enum E_KEY_CODE : uint32_t
+	enum E_KEY_CODE : uint16_t
```

Comment:
you have more than 256 enums?

Reply:
I assumed the overall number of keys on a keyboard potentially may become bigger than 256

Reply:
first find me an OS where that's the case.